### ISSUE TITLE
Request for inference code of Emu2, Seed-Llama and minigpt5

### ISSUE THREAD

**Issue by URRealHero** [HKUST-LongGroup/CoMM#1]
In your paper, I saw you test these three models to do the interleaved format generation task. However, when I'm trying to use these models, they cannot do the interleaved generation.(especially emu2 which officially only support text output, seed-llama's sft model doesn't support interleaved output). So I'm curious about how did you do that. Is there any tricks? Or I should finetune the model?

**Comment by URRealHero** [comment]
I'm really eager to see your finetuned params~

**Comment by chenwei746** [comment]
Hi, Thank you for your interest!
All these three models must be fine-tuned on our dataset for interleaved generation. Emu2 has an image-generation version which supports image output and seed-llama must be fine-tuned on CoMM for interleaved output. Due to the time needed for code organization, I will release the training code and model checkpoints in October.

**Comment by URRealHero** [comment]
Thx a lot, I'm downloading your dataset. So I bet minigpt-5 also requires finetune. I'm making an interleaved benchmark and wanna test these models. If it's possible, would you please email me some of the raw code scripts? because I'm really in trouble of these models' code. Sorry if I offended you. QAQ

**Comment by chenwei746** [comment]
Hi, given your needs, I'll try to organize the model inference code by this weekend. I will release inference code and model checkpoints then.

**Comment by URRealHero** [comment]
Thanks a lot! With my heartfelt thanks for the assistance I've received from you. I'm excited about what your team is set to achieve next and I believe your work will lead to even more remarkable developments!

**Comment by URRealHero** [comment]
Awww! see ur update! looking forward to the others
